Formula/leapp-cli.rb

```diff
@@ -0,0 +1,23 @@
+require "language/node"
+
+class LeappCli < Formula
+  desc "Cloud credentials manager cli"
+  homepage "https://github.com/noovolari/leapp"
+  url "https://registry.npmjs.org/@noovolari/leapp-cli/-/leapp-cli-0.1.13.tgz"
+  sha256 "944ce4928c2d2dd562890310ea6f2f96d3ace5f1c3b7e8bbc9ef8323403359d8"
+  license "MPL-2.0"
+
+  depends_on "python@3.9" => :build
```

Comment:
Does it work with 3.10?

Reply:
It should. Should I drop the `3.9` or use `3.10` ?